types/pdfmake/index.d.ts

```diff
@@ -181,7 +181,7 @@ declare module "pdfmake/build/pdfmake" {
     }
 
     interface TDocumentDefinitions {
-        background?: () => string | string;
+        background?: ((currentPage: number, pageSize: PageSize) => string | Content | null) | string;
```

Comment:
```suggestion
        background?: string | ((currentPage: number, pageSize: PageSize) => string | Content | null);
```

Just a suggestion to clarify this a little bit, since the series of `string | Content | null) | string;` takes a little brainpower to parse.

Reply:
Agreed. I initially only kept the order as it was to make it clearer from the diff what I was changing